taxonomy/models.py

```diff
@@ -50,19 +50,30 @@ class Meta:
     @transaction.atomic
     def save(self, *args, **kwargs):
         """Handle slugs."""
+        create_notset = False
         if self.id is None or self.name != get_object_or_404(
                 Vocabulary, id=self.id).name:
             slug = slugify(self.name)
             count = 1
+            if self.id is None:
+                create_notset = True
             while Vocabulary.objects.filter(slug=slug).exists():
                 slug = "{0}{1}".format(slugify(self.name), count)
                 count += 1
             self.slug = slug
-        return super(Vocabulary, self).save(*args, **kwargs)
+
+        super(Vocabulary, self).save(*args, **kwargs)
+        if create_notset:
+            Term.objects.create(
+                vocabulary=self, label=Term.EMPTY_VALUE, weight=0)
+        return self
 
 
 class Term(BaseModel):
     """Model for term table"""
+
+    EMPTY_VALUE = '--not set--'
```

Comment:
this looks pretty ugly in the interface, but not sure what to use
@pwilkins do you have any suggestion?